TheWarWithin/Priorities/RogueAssassination.simc

```diff
@@ -7,29 +7,31 @@ actions.precombat+=/variable,name=trinket_sync_slot,value=2,if=trinket.2.has_sta
 actions.precombat+=/variable,name=effective_spend_cp,value=cp_max_spend-2<?5*talent.hand_of_fate
 # Pre-cast Slice and Dice if possible
 actions.precombat+=/stealth
-actions.precombat+=/slice_and_dice,precombat_seconds=1,if=refreshable&!buff.indiscriminate_carnage.up
+actions.precombat+=/slice_and_dice
 
 # Restealth if possible (no vulnerable enemies in combat)
 actions+=/stealth
 # Interrupt on cooldown to allow simming interactions with that
 actions+=/kick
-# Conditional to check if there is only one enemy
-actions+=/variable,name=single_target,value=spell_targets.fan_of_knives<2
 # Combined Energy Regen needed to saturate (TODO: estimate poisoned_bleeds better)
-actions+=/variable,name=regen_saturated,value=energy.regen_combined>15
-# Check if we should be using our energy
-actions+=/variable,name=not_pooling,value=(dot.deathmark.ticking|dot.kingsbane.ticking|debuff.shiv.up)|(buff.envenom.up&buff.envenom.remains<=1)|energy.pct>=(40+30*talent.hand_of_fate-15*talent.vicious_venoms)|boss&fight_remains<=20
+actions+=/variable,name=regen_saturated,value=energy.regen_combined>30
```

Comment:
This is a really high value until/unless the addon does a better job of counting poisoned bleeds to forecast those Energy gains.  I'd keep the value lower.

Reply:
ok!